tap-agent/src/agent/sender_account.rs

```diff
@@ -458,6 +463,17 @@ impl Actor for SenderAccount {
                     state.add_to_denylist().await;
                 }
             }
+            SenderAccountMessage::UpdateInvalidReceiptFees(allocation_id, unaggregated_fees) => {
+                state
+                    .invalid_receipts_tracker
+                    .update(allocation_id, unaggregated_fees.value);
+
+                // invalid receipts can't go down
+                let should_deny = !state.denied && state.deny_condition_reached();
+                if should_deny {
+                    state.add_to_denylist().await;
+                }
```

Comment:
:thinking: yes though I guess at some point we should have a process to let the indexer clear the failed receipts

Reply:
Agreed, but this could be done manually by deleting the invalid receipts and restarting tap-agent.

In another case, we could have indexer-cli to communicate with tap-agent and clear the receipts. 

Right now, the only way to update the invalid receipts is by sending a rav request and that rav request contains invalid receipts. We won't have any rav requests if the invalid receipt is the deny cause of a sender.